tuf-spec.md

```diff
@@ -1116,16 +1116,29 @@ repo](https://github.com/theupdateframework/specification/issues).
   cycle, report the potential freeze attack.  On the next update cycle, begin
   at step 0 and version N of the root metadata file.
 
-  * **1.9**. **If the timestamp and / or snapshot keys have been rotated, then
-  delete the trusted timestamp and snapshot metadata files.** This is done in
-  order to recover from fast-forward attacks after the repository has been
-  compromised and recovered. A _fast-forward attack_ happens when attackers
-  arbitrarily increase the version numbers of: (1) the timestamp metadata, (2)
-  the snapshot metadata, and / or (3) the targets, or a delegated targets,
-  metadata file in the snapshot metadata. Please see [the Mercury
+  * **1.9**. **Fast-forward attack recovery** A _fast-forward attack_ happens
+  when attackers arbitrarily increase the version numbers in any of the
+  timestamp, snapshot, targets, or delegated targets metadata. To recover from
+  fast-forward attacks after the repository has been compromised and recovered,
+  certain metadata files need to be deleted as specified in this section.
+  Please see [the Mercury
   paper](https://ssl.engineering.nyu.edu/papers/kuppusamy-mercury-usenix-2017.pdf)
   for more details.
 
+    * **1.9.1**. **Targets recovery** If a threshold of targets keys have been
```

Comment:
You talk about top-level targets below, but not here.  Do you mean to differentiate them?  Does any delegated role's key loss result in the top-level targets file being deleted?

Reply:
I do mean top-level targets keys here too. I thought that might be clear, because root metadata only lists top-level targets keys.

Regarding your other question, why would we delete the top-level targets file if any delegated role's key is lost?  There is a detailed explanation about ffwd-recovery when delegated targets keys have been compromised in the commit message of https://github.com/theupdateframework/specification/pull/86/commits/f50f522b86e92f3f827e7af647ec67df4ac41ce1. Does that answer your question?